### PR TITLE
cake-5067-redo | Rebuilt with lodash/get module

### PR DESCRIPTION
**Pull request by cschucode**
Rebuilt the bundle.

### CODE REVIEW

File: source/utils/environment.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { get } from 'lodash';
+import get from 'lodash/get';
```
Comment (JoshuaRogan) [inline source/utils/environment.js]: Hmm, I thought you just merged this change?

Comment (cschucode) [reply source/utils/environment.js]: Just checked master and it contains this change. However, I don't think the rebuilt bundle was included.

Comment (JoshuaRogan) [reply source/utils/environment.js]: Can you merge master into this branch?

Comment (cschucode) [reply source/utils/environment.js]: Would appear this PR is unnecessary.

**Pull request closed without merging**